src/__tests__/actions/profile/profile.test.js

```diff
@@ -0,0 +1,131 @@
+import Enzyme from "enzyme";
+import React from "react";
+import configureMockStore from "redux-mock-store";
+import EnzymeAdapter from "enzyme-adapter-react-16";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import {
+  IS_UPDATING, PROFILE_LOADED, UPDATE_SUCCESS,
+} from "../../../actions/types";
+import { loadProfile, updateProfile } from "../../../actions/profile/profileActions";
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+const API_HOST_URL = process.env.BASE_URL;
+
+let username;
+let url;
+describe("Profile Action tests", () => {
+  const store = mockStore();
+  beforeEach(() => {
+    username = localStorage.getItem("username00000");
+    url = `${API_HOST_URL}/profiles/${username}`;
+    store.clearActions();
+  });
+  afterEach(() => {
+    username = "";
+    url = "";
+    fetchMock.restore();
+  });
+
+  it("tests if can load  a user profile", () => {
```

Comment:
check the grammar for title of your `it` block

Reply:
Thanks @AmosWels, I have made all file names to camelcase just like the other code files